pkg/operator/ceph/cluster/mgr/mgr_test.go

```diff
@@ -433,3 +436,45 @@ func TestCluster_enableBalancerModule(t *testing.T) {
 		assert.NoError(t, err)
 	})
 }
+
+func TestPrometheusRuleTemplate(t *testing.T) {
+	projectRoot := util.PathToProjectRoot()
+	monitoringPath = path.Join(projectRoot, "/deploy/examples/monitoring/")
+	t.Run("default prometheus rule created successfully from template", func(t *testing.T) {
+		pr, err := templateToPrometheusRule("name", PrometheusRuleTemplatePath)
+		assert.NoError(t, err)
+	out:
+		for _, g := range pr.Spec.Groups {
+			if g.Name == "ceph-mgr-status" {
+				for _, r := range g.Rules {
+					if r.Alert == "CephMgrIsAbsent" {
+						assert.Equal(t, r.For, "5m")
+						assert.Contains(t, r.Expr.StrVal, "rook-ceph")
+						break out
+					}
+				}
+			}
+		}
+	})
+	t.Run("overwrite prometheus rule created successfully from template", func(t *testing.T) {
+		prCustomized := PrometheusRuleCustomized{Alerts: Alerts{CephMgrIsAbsent: CephMgrIsAbsent{For: "1m", Namespace: "openshift-storage"}}}
```

Comment:
Let's keep upstream free of downstream terms
```suggestion
		prCustomized := PrometheusRuleCustomized{Alerts: Alerts{CephMgrIsAbsent: CephMgrIsAbsent{For: "1m", Namespace: "test-ns"}}}
```

Reply:
done